### PR TITLE
Swift 5

### PR DESCRIPTION
**Pull request by stephencelis**
Don't have to merge soon, but wanted to open this.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -27,14 +27,14 @@ let package = Package(
...
-    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .exact("1.2.0")),
+    .package(url: "https://github.com/ianpartridge/swift-backtrace.git", .exact("1.1.0")),
+    .package(url: "https://github.com/pointfreeco/Ccmark.git", .branch("master")),
     .package(url: "https://github.com/pointfreeco/swift-html.git", .exact("0.2.1")),
+    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .branch("swift-5")),
+    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", from: "1.5.0"),
     .package(url: "https://github.com/pointfreeco/swift-tagged.git", .revision("926e8e0")),
-    .package(url: "https://github.com/pointfreeco/swift-web.git", .revision("2c3d440")),
-    .package(url: "https://github.com/pointfreeco/Ccmark.git", .branch("master")),
+    .package(url: "https://github.com/pointfreeco/swift-web.git", .revision("swift-5")),
```
Comment (mbrandonw) [inline Package.swift]: should we merge all the deps in before merging this?

Comment (stephencelis) [reply Package.swift]: Yeah I think so.

File: Sources/PointFree/Environment.swift [hunk 2]
```diff
@@ -18,6 +19,7 @@ public struct Environment {
   public var date: () -> Date = Date.init
   public var envVars = EnvVars()
   public var episodes = { [Episode]() }
+  public var eventLoopGroup: EventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: System.coreCount)
```
Comment (mbrandonw) [inline Sources/PointFree/Environment.swift]: we can revert this right?

Comment (stephencelis) [reply Sources/PointFree/Environment.swift]: Done!

**Comment by stephencelis** [comment]
@mbrandonw I think this is finally ready to go!

File: Package.swift (hunk 1 above)
Comment (stephencelis) [reply Package.swift]: Done!

**Pull request merged**